mods/common/chrome/multiplayer-directconnect.yaml

```diff
@@ -32,7 +32,7 @@ Background@DIRECTCONNECT_PANEL:
 			Width: 10
 			Height: 25
 			Align: Center
-			Text: :
+			Text: Port:
```

Comment:
Not missing. This is on purpose:
![image](https://github.com/OpenRA/OpenRA/assets/756669/0963092e-c3ce-4fc1-ab14-9c9fcb9c4201)

Reply:
<img width="408" alt="Screenshot 2023-10-21 at 19 58 59" src="https://github.com/OpenRA/OpenRA/assets/37534529/d72c5f95-51e1-44f0-8b84-2e23f0f7dd8e">

Reply:
Then you need to change the non-`cnc` layout. Don't just add the string.

Reply:
Yes, fixed